airflow/providers/ftp/hooks/ftp.py

```diff
@@ -115,7 +115,7 @@ def describe_directory(self, path):
             files = dict(mlsd(conn))
         return files
 
-    def list_directory(self, path, nlst=False):
```

Comment:
Is this change backward compatible?

Reply:
This parameter not use in this function

Reply:
So I think we could just remove it directly

Reply:
I am ok with that but then we have to add note in UPDATING.md describing what has changed as this is public interface (someone can use this hook in custom operators).

Reply:
Fix